Using vl2png (and etc bin tools) via npx

```
C:\Sandbox\Nodetest>npx vl2png --help
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/vl2png - Not found
npm ERR! 404
npm ERR! 404  'vl2png@latest' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

Is vl2png et al meant to be recognised by npx as valid aliases? The VL docs suggest they should be.

--- a/site/usage/compile.md
+++ b/site/usage/compile.md
@@ -92,3 +92,7 @@ The `vg2vg` command supports an optional argument `p` that formats the generated
 The easiest way to run these commands is to use [`npx`](https://github.com/npm/npx). `npx` will automatically run the commands from the local `node_modules/.bin`, or from a central cache, installing any packages needed in order for the commands to run. It Is not necessary to have Vega or Vega-Lite installed ahead of time for `npx` to work.
 
 `npx vl2vg spec.vl.json`
+
+### Using npx
+
+To use [npx](https://github.com/npm/npx), a tool to execute npm binaries, provide the required packages and choose the binary you want to run. For example, to run `vl2png`, run `npx -p vega -p vega-lite vl2png --help`.

